src/index.js

```diff
@@ -0,0 +1,34 @@
+import list from './list.js';
+import './style.css';
+import more from './more.png';
+import recycle from './recycle.png';
+import left from './to-left.png';
+
+function component() {
+  list.sort((a, b) => a.index - b.index);
+
+  const container = document.querySelector('.container');
+  const element = document.createElement('div');
+  element.id = 'list_elemnt';
+  element.innerHTML = `<p>Today's To Do</p> <img class="refresh" src="${recycle}" alt="refresh page">`;
+  const input = document.createElement('div');
+  input.id = 'input_div';
+  input.innerHTML = `<input placeholder="Add to your list..."><img class="clear" src="${left}"alt="clear">`;
+  const listObj = document.createElement('div');
+  listObj.id = 'listObjId';
+  list.forEach((lists) => {
+    const listDiv = document.createElement('div');
+    listDiv.id = 'eachList';
+    listDiv.innerHTML = `<div class="left"><input type=checkbox id="check"><span class="des">${lists.description}</span></div><img class="more" src="${more}" alt="more">`;
+    listObj.appendChild(listDiv);
+  });
+  const completed = document.createElement('div');
+  completed.id = 'comp';
+  completed.innerHTML = '<span> clear all completed<span>';
+  container.appendChild(element);
+  container.appendChild(input);
+  container.appendChild(listObj);
+  container.appendChild(completed);
+}
```

Comment:
- [ ] [OPTIONAL] I'd suggest you use ES6 arrow functions instead of the old `function` keyword as it's becoming an industry standard and It allows you to write less code and do more, this will especially show when your codebase grows larger. ✔
- You can refactor the above as:
```
const component = () => {
    . . .
}
```